### PR TITLE
feat(bm-server-components): add ipmi component

### PR DESCRIPTION
**Pull request by radireddy**
ref: MANAGER-6937

Signed-off-by: Ravindra Adireddy <ravindra.adireddy@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/baremetal-server-components`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-6937
| License          | BSD 3-Clause

## Description

Added ipmi tab, ipmi restart and ipmi order components.

### CODE REVIEW

File: packages/manager/modules/bm-server-components/src/index.js [hunk 1]
```diff
@@ -3,11 +3,31 @@ import bmServerTechnicalDetailsTile from './technical-details';
...
 export default {
   bmServerGeneralInfoTile,
   bmServerTechnicalDetailsTile,
   bmServerNetworkTile,
   bmServerEditName,
   bmServerSupportTile,
+  bmServerIpmi,
+  bmServerIpmiRestart,
+  bmServerIpmiOrderKvm,
+  pollingService,
```
Comment (zakariasahmane) [inline packages/manager/modules/bm-server-components/src/index.js:32]: Why two exports?

File: packages/manager/modules/bm-server-components/src/ipmi/order-kvm/template.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                    value="{{ option.duration }}"
+                    data-ng-model="$ctrl.order.durationSelected"
+                />
+                <span>{{ option.duration | wucDuration:'longDate' }}</span>
+                <strong
+                    class="text-danger float-right"
+                    data-ng-bind-html="option.prices | ducPrice:$ctrl.user.ovhSubsidiary"
+                >
+                </strong>
+                <!-- <ovh-manager-catalog-price
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/ipmi/order-kvm/template.html:31]: Please remove commented code

File: packages/manager/modules/bm-server-components/src/ipmi/order-kvm/template.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            data-ng-repeat="option in $ctrl.order.details track by $index"
+        >
+            <oui-field>
+                <input
+                    type="radio"
+                    name="durationSelected"
+                    value="{{ option.duration }}"
+                    data-ng-model="$ctrl.order.durationSelected"
+                />
+                <span>{{ option.duration | wucDuration:'longDate' }}</span>
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/ipmi/order-kvm/template.html:25]: ` <span data-ng-bind="option.duration | wucDuration:'longDate' "></span>`

File: packages/manager/modules/bm-server-components/src/ipmi/order-kvm/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "bm_server_ipmi_order_kvm": "Commander un KVM",
+  "bm_server_kvm_order_step1_title": "Choisissez la durée",
+  "bm_server_kvm_order_step2_title": "Commande d'un KVM sur le serveur <em>{{serverName}}</em>",
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/ipmi/order-kvm/translations/Messages_fr_FR.json:4]: Can move `<em>` tag to html instead

File: packages/manager/modules/bm-server-components/src/ipmi/template.html [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+    </div>
+
+    <div data-ng-if="$ctrl.ipmi.isActivated() && $ctrl.disable.testActive">
+        <ul class="list-unstyled">
+            <li>
+                <oui-spinner
+                    data-ng-if="$ctrl.httpState.isLoading()"
+                    data-size="s"
+                ></oui-spinner>
+                <span
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/ipmi/template.html:265]: `aria-hidden="true"`

File: packages/manager/modules/bm-server-components/src/ipmi/template.html [hunk 6]
```diff
@@ -0,0 +1,330 @@
...
+                    class="oui-icon oui-icon-error text-danger"
+                    data-ng-if="$ctrl.httpState.isError()"
+                >
+                </span>
+                <span
+                    class="oui-icon oui-icon-success text-success"
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/ipmi/template.html:271]: `aria-hidden="true"`

Comment (varun257) [reply packages/manager/modules/bm-server-components/src/ipmi/template.html:271]: Same for all icons in this page

File: packages/manager/modules/bm-server-components/src/ipmi/order-kvm/translations/Messages_fr_FR.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  "bm_server_kvm_order_step2_duration": "Période :",
+  "bm_server_kvm_order_step2_details": "Détails de la commande :",
+  "bm_server_kvm_order_step2_price": "Prix :",
+  "bm_server_kvm_order_finish_success": "La commande numéro <a href='{{orderUrl}}' target='_blank'>{{orderId}}</a> a bien été prise en compte.",
```
Comment (varun257) [inline packages/manager/modules/bm-server-components/src/ipmi/order-kvm/translations/Messages_fr_FR.json:8]: `rel="noopener"` for anchor tag

**Review by varun257 — APPROVED** [review]

**Pull request merged**